trywhatyoubuildDEVMODE-beg.adoc

```diff
@@ -0,0 +1,11 @@
+=== Try what you'll build
+
+The `finish` directory in the root of this guide contains the finished inventory application. Give it a try before you proceed.
+
+To try out the application, first go to the `finish` directory and run the following
+Maven goal to build the application and deploy it to Open Liberty:
+
+[role='command']
+```
+mvn liberty:run
+```
```

Comment:
If all are good, can add the following for this file: 

After you see the following message, your application server is ready.

[role="no_copy"]
----
The defaultServer server is ready to run a smarter planet.
----